deploy/packaging/alpine/build_apks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+pckdir=$(dirname $(realpath $0))
+cd /workspace
+signkeys="/sign_keys/mdsplus@mdsplus.org-589e05b6.rsa"
+if [ -f $signkeys ]
+then
+  mkdir -p ~/.abuild
+  echo -e "PACKAGER_PRIVKEY=\"$signkeys\"\n" > ~/.abuild/abuild.conf
+else
+  abuild-keygen -aqn # generate new keys
+fi
+# we have to unset srcdir as it is a reserved name in abuild
+# in the process of building the apks it will wipe $srcdir
+srcdir= \
```

Comment:
I'm confused by the \ here, are those intentional?

Reply:
it ties the lines together as if there were no new line.
```sh
A=a B=a cmd
```
wil run cmd with envA set to a and env B set to b without modifying the env of the caller